.bingo/Variables.mk

```diff
@@ -88,12 +88,6 @@ $(JSONNETFMT): .bingo/jsonnetfmt.mod
 	@echo "(re)installing $(GOBIN)/jsonnetfmt-v0.16.0"
 	@cd .bingo && $(GO) build -modfile=jsonnetfmt.mod -o=$(GOBIN)/jsonnetfmt-v0.16.0 "github.com/google/go-jsonnet/cmd/jsonnetfmt"
 
-LICHE := $(GOBIN)/liche-v0.0.0-20200229003944-f57a5d1c5be4
```

Comment:
You need to also remove `.bingo/liche.mod` to make it permanent.

Reply:
Seems you need enable new github action in pr, details refer to [this](https://docs.github.com/en/github/administering-a-repository/disabling-or-limiting-github-actions-for-a-repository), there is not a github action named as "Check Markdown links".

Reply:
ack

Reply:
It looks like it's enabled:

![image](https://user-images.githubusercontent.com/6950331/91971974-ecb91180-ed11-11ea-81af-44964edc7f16.png)

Reply:
Now all work!